These tests were dropped upstream in version 10.1.0 and are arguable broken
before that.

diff --git a/test/chrono-test.cc b/test/chrono-test.cc
index 33c0c2c6653e..f047c1ea90c8 100644
--- a/test/chrono-test.cc
+++ b/test/chrono-test.cc
@@ -730,9 +730,6 @@ TEST(chrono_test, special_durations) {
             "03:33");
   EXPECT_EQ(fmt::format("{:%T}", std::chrono::duration<char, std::mega>{2}),
             "03:33:20");
-  EXPECT_EQ("44.000000000000",
-            fmt::format("{:%S}", std::chrono::duration<float, std::pico>(
-                                     1.54213895E+26)));
 }
 
 TEST(chrono_test, unsigned_duration) {
diff --git a/test/format-test.cc b/test/format-test.cc
index baf33569f96d..52ac2cafeade 100644
--- a/test/format-test.cc
+++ b/test/format-test.cc
@@ -1427,7 +1427,6 @@ TEST(format_test, prettify_float) {
   EXPECT_EQ("12.34", fmt::format("{}", 1234e-2));
   EXPECT_EQ("0.001234", fmt::format("{}", 1234e-6));
   EXPECT_EQ("0.1", fmt::format("{}", 0.1f));
-  EXPECT_EQ("0.10000000149011612", fmt::format("{}", double(0.1f)));
   EXPECT_EQ("1.3563156e-19", fmt::format("{}", 1.35631564e-19f));
 }
 